### DISCUSSION TITLE
I built a GitHub Action to add missing alt text to images defined in markdown formatted text

### DISCUSSION THREAD

**Discussion by jonathanalgar** [community/community#92338] [Actions]
### Select Topic Area

Show & Tell

### Body

In case anyone is interested,

Demo: https://github.com/jonathanalgar/docs-demo/pull/1

Powered by:

* https://github.com/jonathanalgar/alttexter-ghclient
* https://github.com/jonathanalgar/alttexter

Take care all.

**Comment by jonathanalgar** [comment]
@inkblotty Hi Katie, I came across your and your team's great work on [accessibility-alt-text-bot](https://github.com/github/accessibility-alt-text-bot) when I was searching for alt-text related projects on GitHub. I note through the [emoji](https://github.com/marketplace/actions/accessibility-alt-text-bot#contributors-) you are open to feedback and suggestions. Did you ever consider adding (optional) functionality to call a multimodal language model to provide suggestions for the alt-text? I did something like this for documentation in the context of a pull request (see demo above) and the results were very interesting. Thanks, J

**Reply by inkblotty** [reply]
Very cool @jonathanalgar! I'm going to tag in our active maintainers to the bot @khiga8 @kendallgassner 🙏 

Having active suggestions while authoring is a different model that we've discussed :) That could also be a good place for these types of suggestions. I've tagged in some people internally.

**Comment by jonathanalgar** [comment]
> Having active suggestions while authoring is a different model that we've discussed :)

@inkblotty awesome! although I've found _post-authoring_ can be quite powerful as well as the model can pull in (complete) relevant textual context as well as the image(s).

ICYI I recently gave a talk where I spoke through the [demo](https://github.com/jonathanalgar/docs-demo/pull/1) and shared some thoughts on the UX: https://www.youtube.com/watch?v=jy5RGdJwSz8&t=103s

Would really value your feedback on how it could be improved!
